add clarification slack job notifis (#8489)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/job-notifications.md

## Slack notifications (account) <Lifecycle status="private_beta" />  {#slack-notifications-account}
:::info
Configuring Slack notifications at the account level is currently available in private beta. To request access, contact your account manager. Only refer to these instructions if you have access to the private beta feature.
:::

Integrate Slack with <Constant name="dbt_platform" /> at the account level to receive job notifications in Slack. dbt integrates with Slack via OAuth to ensure secure authentication. 

A single <Constant name="dbt_platform" /> account can integrate with one Slack workspace. 

### Prerequisites

- You have a Slack workspace that you want to receive job notifications from.
- A <Constant name="dbt_platform"/> account admin must link the Slack app at the account level.
- To install the Slack app to a workspace, your Slack org must permit app installations. In some orgs this requires a Slack admin approval.

> After an account admin links the Slack app for the account, [any licensed user](/docs/cloud/manage-access/seats-and-users) in the account can configure Slack job notifications so long as they are assigned to a [group](/docs/cloud/manage-access/about-user-access#groups) with **Account Admin**, **Owner**, or **Member** permissions. IT licenses don't have access to configure Slack job notifications.

Reviewer: ```suggestion
After an account admin links the Slack app for the account, [any licensed user](/docs/cloud/manage-access/seats-and-users) assigned to a [group](/docs/cloud/manage-access/about-user-access#groups) with **Account Admin**, **Owner**, or **Member** permissions can configure Slack job notifications. Users with IT licenses do not have access to configure Slack job notifications.
```
